### DISCUSSION TITLE
Tamanho scrollbar

### DISCUSSION THREAD

**Discussion by thauealfredo** [frontendbr/forum#1960] [Perguntas e Respostas]
Como eu consigo mudar o tamanho da scrollbar apenas com CSS, tem como ?

**Comment by ninetails** [comment]
O site [CSS Tricks](https://css-tricks.com/) possui bastante material sobre (mas em inglês infelizmente).

- [The Current State of Styling Scrollbars](https://css-tricks.com/the-current-state-of-styling-scrollbars/)
- [Custom Scrollbars in WebKit](https://css-tricks.com/custom-scrollbars-in-webkit/)

Acho que tinha um pra todos os navegadores mas não estou achando.

**Reply by thauealfredo** [reply]
![image](https://user-images.githubusercontent.com/50925608/103023145-249bcd00-452c-11eb-827c-f9c92cf87b5b.png)
 eu achei esses trechos, porem quando eu mexo nesse 3° trecho, muda o tamanho da espaço da scrollbar, que acaba mudando o tamanho dela. Mas ai ela começa no meio da div e n vai ate o final

**Reply by ninetails** [reply]
tem como subir no [Codepen](https://codepen.io/) se for algo aberto?
